ansible/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+- name: "Build hosts file"
+  lineinfile: dest=/etc/hosts regexp='.*{{ item }}$' line="{{item}}" state=present
```

Comment:
Is the purpose of that `regexp` to uncomment `{{ item }}` if it exists and is commented out?